pom.xml

```diff
@@ -37,6 +37,7 @@
     <excludedIntegrationTests/>
 
     <junixsocket.version>2.9.0</junixsocket.version>
+    <opentelemetry.exporter.version>0.27.0</opentelemetry.exporter.version>
```

Comment:
There's unfortunately a bug in the documentation (or implementation?) of this library. It claims to support Java 8, but it actually doesn't. That is why the tests are currently failing for all Java 8 builds. That is the reason that PGAdapter is using an older version of the library.

Reply:
Okay. Let me to use an older version then.